playground/optimize-deps/nested-exclude/package.json

```diff
@@ -4,6 +4,6 @@
   "version": "1.0.0",
   "main": "index.js",
   "dependencies": {
-    "nested-include": "link:./nested-include"
```

Comment:
This should be a `file:` protocol all along, but it wasn't needed before because this package (nested-include) was depended as a `file:` protocol too which shadowed this part here.


I also moved `nested-include` out one level. It feels weird to have it inside `nested-exclude`. Shouldn't affect test logic too.